src/components/AttentionBox/__tests__/__snapshots__/attentionBox-snapshot-tests.jest.tsx.snap

```diff
@@ -366,58 +366,47 @@ exports[`AttentionBox renders correctly renders correctly with onClose 1`] = `
   <div
     className="closeIconWrapper wrapper"
   >
-    <span
```

Comment:
CI passed in their PR?

Reply:
No, but I thought that I'll fix it by myself